autopts/ptsprojects/zephyr/tmap.py

```diff
@@ -92,14 +92,44 @@ def test_cases(ptses):
         TestFunc(btp.vcs_register, 1, False, 100),
         TestFunc(btp.core_reg_svc_tbs),
         TestFunc(btp.core_reg_svc_csip),
+        TestFunc(btp.core_reg_svc_mics),
+        TestFunc(btp.core_reg_svc_aics),
+        TestFunc(btp.core_reg_svc_vocs),
+        TestFunc(btp.core_reg_svc_gmcs),
+        TestFunc(btp.core_reg_svc_cas),
+        TestFunc(btp.core_reg_svc_micp),
+        TestFunc(stack.aics_init),
         TestFunc(stack.ascs_init),
         TestFunc(stack.bap_init),
         TestFunc(stack.cap_init),
+        TestFunc(stack.csip_init),
+        TestFunc(stack.gmcs_init),
+        TestFunc(stack.micp_init),
+        TestFunc(stack.mics_init),
+        TestFunc(stack.pacs_init),
+        TestFunc(stack.tbs_init),
         TestFunc(stack.tmap_init),
         TestFunc(stack.vcp_init),
         TestFunc(stack.vcs_init),
-        TestFunc(stack.tbs_init),
-        TestFunc(stack.csip_init),
+        TestFunc(stack.vocs_init),
+    ]
+
+    pixit_param_bms_ctxt_bv_01_c = [
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_TARGET_ASE_CHARACTERISTIC", "SINK_ASE")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_Codec_ID", "0600000000")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_Codec_Specific_Configuration", "0001")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_QoS_Framing", "UNFRAMING")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_QoS_PHY", "2M_PHY")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_QoS_SDU_Interval", "64")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_QoS_Max_SDU", "10000")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_QoS_Retransmission_Number", "2")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_QoS_Max_Transport_Latency", "40")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_QoS_Presentation_Delay", "40000")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_Company_ID", "0000")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_VS_Codec_ID", "0000")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_broadcast_code", "0102680553F1415AA265BBAFC6EA03B8")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_Sync_Timeout", "20000")),
+        TestFunc(lambda: pts.update_pixit_param("TMAP", "TSPX_STREAMING_DATA_CONFIRMATION_METHOD", "By Playing")),
```

Comment:
Are any of these actually necessary for that test? Looking at the test spec, none of these values matter.